software/bit.yml

```
@@ -0,0 +1,11 @@
+name: bit
+website_url: https://hub.docker.com/r/sjdonado/bit
```

```suggestion
website_url: https://github.com/sjdonado/bit
```
I don't see any additional value in adding the docker hub package, which does not have any description of the project what so ever, especially since it is already linked in your project's readme.